shared/actions/fs/index.js

```diff
@@ -477,6 +479,17 @@ function* loadResets(action: FsGen.LoadResetsPayload): Saga.SagaGenerator<any, a
   yield Saga.call(resetRpc.run)
 }
 
+function letResetUserBackIn(action: FsGen.LetResetUserBackInPayload) {
```

Comment:
in actions we prefer arrows for all non generators since they can be more concise.